core/templates/components/shared-component.module.ts

```diff
@@ -145,8 +150,11 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     MaterialModule,
     NgbTooltipModule,
     // Components, directives, and pipes.
+    AngularHtmlBindWrapperDirective,
```

Comment:
No need to add it here. This is already exported as a part of DynamicContentModule

Reply:
Done